wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -14,7 +14,10 @@ extra-source-files:  README.md
 cabal-version:       >=1.10
 
 library
-  hs-source-dirs:      src
+  hs-source-dirs:      src server
```

Comment:
If we're going to do this I want to move all the `server` code into `src`. Can happen at a later date/PR.